modules/apps/map/map-common/test/liferay/MapBase.es.js

```diff
@@ -29,6 +37,8 @@ describe('MapBase', () => {
 			});
 		}
 
+		_getControlsConfig() {}
```

Comment:
Same here, why are you overriding?

Reply:
@bryceosterhaus without this override, the test is complaining:
```
    TypeError: Cannot read properties of undefined (reading 'map')

      289 |     _getControlsConfig() {
      290 |             const config = {};
    > 291 |             const availableControls = this.controls.map((item) => {
          |                                                     ^
      292 |                     return typeof item === 'string' ? item : item.name;
      293 |             });
      294 | 
```

I thought it was fine, because the test already overrides `setCenter() {}`, I'll try to remove it and find a better solution